anatomy/tracks/internal-linking.md

```diff
@@ -0,0 +1,24 @@
+# Internal Linking
+
+Links rendered via Exercism's Markdown parser are checked to see if they are internal, and if so may be decorated with tooltips.
+
+A tooltip for a concept might look like this:
+
+![concept-tooltip](../../.imgs/concept-tooltip.png)
+
+## Usage
+
+There is no special syntax for links to be decorated in this manner.
+Instead we check to see if the link href follows one of two forms:
+
+- A URL. e.g. https://exercism.io/tracks/ruby/concepts/basics
+- An absolute path. e.g. /tracks/ruby/concepts/basics
+
+We do not currently support relative URLs (e.g. those beginning with `./`) because the markdown parser is not aware of the context of the document it renders, the context it is rendering in, or the structure of a git repository.
```

Comment:
```suggestion
We do not currently support relative URLs (e.g. those beginning with `./` or `../`) because the Markdown parser is not aware of the context of the document it renders, the context it is rendering in, or the structure of a git repository.
```